sbi/inference/potentials/likelihood_based_potential.py

```diff
@@ -172,6 +219,78 @@ def _log_likelihoods_over_trials(
     return log_likelihood_trial_sum
 
 
+def _log_likelihood_over_iid_conditions(
+    x: Tensor,
+    theta_without_condition: Tensor,
+    condition: Tensor,
+    estimator: ConditionalDensityEstimator,
+    track_gradients: bool = False,
+) -> Tensor:
+    """Returns $\\log(p(x_o|\theta, condition)$, where x_o is a batch of iid data, and
```

Comment:
More general isnt it that  $\prod_{i=1}^Np(x_i|condition_i, \theta)$.

In this case the difference between "theta" and "condition" is that "theta" is a *global variable* and "condition" is a *local variable*, no? 

Not sure if "condition" is the best word to use here (might be something domain specific). Wouldnt "local_theta" and "global_theta" make more sense?

Reply:
Yes, I like the distinction between local and global theta, thanks! changed the naming accordingly.